subprojects/packagefiles/liburing/meson.build

```diff
@@ -0,0 +1,113 @@
+project('liburing', ['c','cpp'],
+        version: 2.0,
```

Comment:
This can't be a float, needs to be a string or meson immediately errors out. Did this work for you at some point?

Reply:
No. Seams I have pushed a outdated version. Sorry for that!